bitbots_quintic_walk/config/walking_wolfgang_viz.yaml

```diff
@@ -74,7 +74,7 @@ walking:
     engine_freq: 100.0
 
     # parameters for bioIK
-    bio_ik_time: 0.01
```

Comment:
This parameter should probable be renamed to ik_timeout here and in the code too.

Reply:
correct